### ISSUE TITLE
Payment received stamp settings are incorrect.

### ISSUE THREAD

**Issue by pinalj** [TycheSoftwares/Print-Invoice-Delivery-Notes-for-WooCommerce#264]
**Describe the bug**
The payment received stamp settings currently present in the plugin do not match the BRD.

**To Reproduce**
1. Go to WooCommerce > Settings > Print > Documents > Receipts
2. Click on 'Customize'
3. Scroll down to Payment Received stamp
4. See the difference

**Expected behavior**
It should be the same as the BRD

**Screenshots or Video**
BRD: https://app.screencast.com/9j9nOOY9biQRA
Actual setting: https://app.screencast.com/h7hxrBXSy4Szi

**Comment by apeksha10** [comment]
Re-open this issue. As the above issue is still replicated

**Comment by gaminikhil** [comment]
Yes. But we can consider this in the next update or we can wait for the client's requirements.

**Comment by apeksha10** [comment]
Discussed in group. Moving this issue to the next milestone.

**Comment by handelce** [comment]
This is no longer tenable as we have v7 where the UI has been replaced with React.
